cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -717,7 +719,11 @@ public MapAndStatus(T loadMap, K status) {
             return onNew(reconciliation, host, apiClient, kafkaRebalance, rebalanceOptionsBuilder);
         } else {
             // Stay in the current NotReady state, returning null as next state
-            return Future.succeededFuture(new MapAndStatus<>(null, null));
+            Set<Condition> conditions = validate(reconciliation, kafkaRebalance);
+            if (rebalanceAnnotation != KafkaRebalanceAnnotation.none) {
```

Comment:
Can we no apply a `Refresh` annotation to a `NotReady` KafkaRebalance? Is that not valid / no-op?